index.html

```diff
@@ -2596,30 +2596,38 @@ <h3>Data Schemas</h3>
       </section>
 
       <section>
-        <h2>Context Integrity</h2>
+        <h2>External Resource Integrity</h2>
         <p>
-          In some cases it is desirable to know that the contents of the
+          When including a link to an external resource in a VC it is 
+          desireable to know if the resource that is pointed to, is the
+          same at signing time, as at verification time.  This applies to
+          both cases where there is an external resource that is 
+          remotely retrieved, as well as to cases where the issuer and/or
+          verifier may have local cached copies of a resource.
+        </p>
+        <p>
+          It is also desirable to know that the contents of the
           context(s) utilized in the verifiable credential are the same as
-          used by both the issuer and verifier.  
+          used by both the issuer and verifier. 
         </p>
         <p>
-          To validate that a context included in a Verifiable Credential is
+          To validate that a resource referenced by a Verifiable Credential is
           the same at verification time as at issuing time an implementer
-          MAY include a property named <code>contextIntegrity</code> that
+          MAY include a property named <code>resourceIntegrity</code> that
           stores an array of objects that describe additional integrity
-          metadata about each context used by the VC. If
-          <code>contextIntegrity</code>
+          metadata about each resource referenced by the VC. If
+          <code>resourceIntegrity</code>
           is present there MUST be an object in the array for each remote
-          context.
+          resource.
         </p>
         <p>
           Each object in the
-          <code>contextIntegrity</code> array MUST contain the following:
-          the URL to the context named <code>context</code>, a
+          <code>resourceIntegrity</code> array MUST contain the following:
+          the URL to the resource named <code>resource</code>, a
           <code>timestamp</code>
           that indicates the time at which the hash was computed, the
           <code>hash</code>
-          of the context, and the <code>method</code> which indicates what
+          of the resource, and the <code>method</code> which indicates what
           hashing algorithm was used as listed as the 'Hash Name String'
           property from the <a
```

Comment:
```suggestion
          of the resource;
          and the <code>method</code> which indicates what
          hashing algorithm was used via the string listed as the
          'Hash Name String' property in the <a
```

Reply:
This might could use some further massaging, after these changes are applied, so the whole block is green (no more red lines breaking the stream).

Reply:
this property has been remove in favor of embedding the hash alg SRI style